lua/asllib.lua

```diff
@@ -11,6 +11,16 @@ function negate( v )
     else return -v end
 end
 
+function n2v( n )
+	if type(n) == 'function' then
+		return function () return n()/12 end
+	else return n/12 end
+end
+
+function note( noteNum, duration )
+	return{to(n2v(noteNum), 0), to('here', duration)}
```

Comment:
could you update the layout here to make it match the rest of asllib calls. something like:
```
return{ to( n2v(noteNum), 0 )
      , to( 'here', duration )
      }
```
i think it makes it clearer that it's a chaining of 2 `to`s.